packages/server/src/course/ical.service.ts

```diff
@@ -49,7 +49,7 @@ export class IcalService {
 
     const officeHoursEventRegex = /\b^(OH|Hours)\b/;
 
-    const filteredOfficeHours = officeHours.filter(event =>
+    const filteredOfficeHours = officeHours.filter((event) =>
```

Comment:
we all gotta get our VSCodes in line for this shit

Reply:
yeah feels bad, i think i just need to configure my vscode